lib/automaticupgrades/config.go

```diff
@@ -78,10 +78,10 @@ func GetUpgraderVersion(ctx context.Context) string {
 		out, err := exec.CommandContext(ctx, teleportUpgradeScript, "version").Output()
 		if err != nil {
 			log.WithError(err).Debug("Failed to exec /usr/sbin/teleport-upgrade version command.")
-		} else {
-			if version := strings.TrimSpace(string(out)); version != "" {
-				return version
-			}
+			return ""
+		}
+		if version := strings.TrimSpace(string(out)); version != "" {
+			return version
```

Comment:
Is there a situation where we actually expect a successful execution of teleport-upgrade but no output? If that's also an error condition, shouldn't we want to signal that somehow?

Reply:
Good point. We should ensure semver and log a warning if version output is unexpected.